src/storage/sequelize.ts

```diff
@@ -16,6 +16,9 @@ export type ModelClass = {
 	findAll(options?: {}): Promise<any[]>;
 	create(options: {}): Promise<void>;
 	destroy(options: {}): Promise<void>;
+	dialect: {
+		name: string;
+	};
```

Comment:
You'll need to add this to SequelizeType I think. Did that give different errors?

Reply:
Sorry, fixed the typo